### DISCUSSION TITLE
Surface remeshing - question

### DISCUSSION THREAD

**Discussion by jakisnicknagh** [nschloe/pygalmesh#150] [Q&A]
Hi,
Im working on pygalmesh 7.2 from PIP, but it shouldn't be a problem. 

I'm trying to understand, what's happening with my mesh after remeshing. 
If we look at lions head mesh is not "watertight", why it's happening and how to avoid it/control it/repair it ? 

Im preparing mesh for FEM analysis, so I could make some quick repairs in SpaceClaim and its "ok", but after all I don't get why this problem rises, and how to avoid it ? 

Because after all triangles are where they should be, only they are marked as "holes"(red ones).
Are they "flipped" or something ? 

I will be grateful for explanation,

![image](https://user-images.githubusercontent.com/55050930/123938382-5be5ad80-d997-11eb-8563-a876c73d7182.png)

Best regards,

**Comment by nschloe** [comment]
How is the mesh not watertight?
